### PR TITLE
GCW-3071 Button is working and displayed Stock: The 'x' button to clear the offer search field is displayed in the wrong place fix

### PR DESCRIPTION
**Pull request by shreyas098**
### Ticket Link: 
https://jira.crossroads.org.hk/browse/GCW-3071

### What does this PR do?

Displays the X in correct place both in mobile as well as web app
Screen Shot 2020-03-04 at 5 47 22 PM
Screen Shot 2020-03-04 at 5 48 11 PM

### CODE REVIEW

**Comment by abhinav098** [comment]
@shreyas098 , have you checked on medium screens ? 
i.e. I pads etc.

File: app/templates/components/goodcity/offers-search-overlay.hbs [hunk 1]
```diff
@@ -6,21 +6,24 @@
...
-
-          {{#if searchText}}
-            <i class="pinned-right" {{action 'clearSearch'}}>{{fa-icon 'times-circle' size='lg'}}</i>
-          {{/if}}
-        </div>
-        {{#if isMobileApp}}
-          <div class="scan-button scanner">
-            {{scan-barcode-button onScanComplete=(action "setScannedSearchText")}}
+          <div class="row">
+            <div class="small-10 columns">
```
Comment (abhinav098) [inline app/templates/components/goodcity/offers-search-overlay.hbs]: I think you can keep 12 columns for large screen as we don't have scanner on the large screens. i.e. on web browsers or computers.

**Comment by shreyas098** [comment]
> @shreyas098 , have you checked on medium screens ?
> i.e. I pads etc.

File: app/templates/components/goodcity/offers-search-overlay.hbs (hunk 1 above)
Comment (shreyas098) [reply app/templates/components/goodcity/offers-search-overlay.hbs]: Sure

Comment (abhinav098) [reply app/templates/components/goodcity/offers-search-overlay.hbs]: Also specify columns for medium screen as well.

Comment (shreyas098) [reply app/templates/components/goodcity/offers-search-overlay.hbs]: ok

**Pull request merged**